README.md

```
@@ -1,19 +1 @@
-`sudo apt install npm`
-
-
-`npm install`
-
-`node app.js`
-
-or Run by docker compose
-
-test
-
+# Node todo application
```

Markdownlint (0.37.0)

Files should end with a single newline character

(MD047, single-trailing-newline)